forc/src/ops/forc_deploy.rs

```diff
@@ -99,11 +103,14 @@ fn create_contract_tx(
 
     let salt = Salt::new([0; 32]);
     let static_contracts = vec![];
-    let storage_slots = vec![];
 
     let contract = Contract::from(compiled_contract);
     let root = contract.root();
-    let state_root = Contract::default_state_root();
+
+    // The VM requires that storage slots are sorted.
```

Comment:
Technically, there's no requirement that the JSON ABI is sorted. Would be nice to add that this step isn't strictly necessary and not guaranteed, and downstream tooling should sort it themselves.

Reply:
I created this issue: https://github.com/FuelLabs/fuel-tx/issues/153. 

If I don't currently sort the slots, I get a VM error about the slots not being sorted 🤷